tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontend/pages/LandingPage.java

```diff
@@ -16,8 +20,20 @@ public LandingPage(WebDriver driver) {
     //Actions:
     public String verifyLandingPageTitle() {
         WebDriverWait wait = new WebDriverWait(driver, 40);
-        wait.until(ExpectedConditions.titleIs("eFiling Frontend"));
+        wait.until(ExpectedConditions.titleIs("eFiling Frontend Demo App"));
```

Comment:
is this referring to the demo client or the actual frontend?
let's not use `demo` and `frontend` together in same context as it can be quite confusing

if demo, call it `eFiling Demo Client`
if frontend, call it `eFiling Frontend`

Reply:
Page title was used from the actual page itself. Updated the html in the frontend code to have the page title as 'eFiling Demo Client'

Reply:
it was @sdevalapurkar complaining about his own naming ;-)